src/exp_log_geo.jl

```diff
@@ -30,7 +34,7 @@ The result is saved to `q`.
 See also [`exp`](@ref).
```

Comment:
one could add a small technical note here as well, that the most general one to implement is the scaled one?

Reply:
I've added a note, is that enough?